### DISCUSSION TITLE
Nature renderer not painting details with colliders?

### DISCUSSION THREAD

**Discussion by HellbourneAL** [visualdesigncafe/nature-renderer#83] [Help]
Hi, I have 2 prefabs with sphere colliders on them, when I manually place the prefabs in the world the colliders work with my player, when I paint them in the world with nature renderer as details the colliders dont work??

**Comment by zertrax** [comment]
I just had the same problem but using mesh colliders. Replacing them with Box Colliders fixed the problem
Also tested capsule and sphere colliders and they both worked fine, hope this helps

**Comment by maxplooi** [comment]
Colliders are ignored for detail objects, they are only supported for trees and are handled by Unity's default terrain system. You can follow Unity's documentation for how to set them up.
